scripts/libme.sh

```diff
@@ -0,0 +1,7 @@
+download_and_extract https://github.com/NT-Bourgeois-Iridescence-Technologies/MElib/archive/master.zip MElib
```

Comment:
This should be MElib-master, at the end, since that's where it will be extracted.

Reply:
This directory will be different depending on the archive. I would recommend using the tar.gz, which can be used by replacing zip with tar.gz, since the result will be more predictable.

Reply:
Fixed it to MElib v1.1. I used the tar.gz